artemis/config.py

```diff
@@ -698,6 +698,29 @@ class Nuclei:
                 "NUCLEI_TEMPLATE_CHUNK_SIZE is 200, three calls will be made with 200 templates each.",
             ] = get_config("NUCLEI_TEMPLATE_CHUNK_SIZE", default=200, cast=int)
 
+        class PlaceholderPagesHtmlElements:
```

Comment:
IMO add a configuration variable to enable this behavior, default false - by default people may not want random webpages to be skipped